app.json

```
@@ -1,5 +1,5 @@
 {
   "appName": "eigen",
-  "version": "8.0.0",
+  "version": "8.0.1",
```

```suggestion
  "version": "8.0.2",
```

since this is main, lets inc one more.